Modules/Shop/Http/Services/ShopService.php

```diff
@@ -859,18 +854,19 @@ public function searchResult(object $request): array
             ->orWhere('main.title', 'like', '%'.$request->search.'%' )
             ->orWhere('sub.title', 'like', '%'.$request->search.'%' )
             ->orWhere('child.title', 'like', '%'.$request->search.'%' )
-            ->get();;
+            ->get();
         if (!empty($results)) {
             foreach ($results as $resultedProduct) {
                 $brandDetails = Brand::where('user_id', $resultedProduct->user_id)->first();
```

Comment:
get the userIds of all the prouducts and then get all the brands outside the loop in allBrands variable using whereIn and then inside the loop get the required brand form $allBrands.